tests/queries/0_stateless/02716_create_direct_dict_with_lifetime_crash.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE IF NOT EXISTS dict_source (key UInt64, value String) ENGINE=MergeTree ORDER BY key;
```

Comment:
The test name is not good.

The "crash" means abnormal program termination; this is a strong word. If ClickHouse crashes, it is a disaster, requiring immediate attention and stopping all other work across the company. An error message for a query is not "crash".